src/components/Player.vue

```diff
@@ -104,6 +104,9 @@
         :class="{ active: isMenuOpen }"
       >
         {{ player.name }}
+        <div class="pronouns" v-if="player.pronouns && player.pronouns !== ''">
```

Comment:
Either this has to move outside of `div.name` or the `player.name` needs to be put into their own container for styling reasons. (see comment in the CSS part)